app/controllers/api/v0/apps_controller.rb

```diff
@@ -188,8 +188,8 @@ def update
           key :'$ref', :App
         end
       end
-      extend Api::V0::SwaggerResponses::AuthenticationError
-      extend Api::V0::SwaggerResponses::ForbiddenError
+      extend Api::V0::SwaggerResponses::AuthenticationErrorToken
+      extend Api::V0::SwaggerResponses::ForbiddenErrorToken
       extend Api::V0::SwaggerResponses::NotFoundError
       extend Api::V0::SwaggerResponses::ServerError
     end
```

Comment:
FYI, I got tired of all the identical extends and did this: https://github.com/a15k/mothership/blob/master/app/helpers/api/swagger_responses.rb
which slims the controllers down to something like:
https://github.com/a15k/mothership/blob/master/app/controllers/api/v1/users_controller.rb#L15-L17